Composer/packages/extensions/adaptive-form/src/components/fields/EditableField.tsx

```diff
@@ -11,6 +11,7 @@ interface EditableFieldProps extends FieldProps {
   fontSize?: string;
   styles?: Partial<ITextFieldStyles>;
   transparentBorder?: boolean;
+  ariaLabel?: string;
```

Comment:
I'm leaning towards making this a required label. what do you think?

Reply:
I agree it should be, but trying to do that causes all kinds of type-checking issues. I think that should be its own PR later on.